nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -141,7 +142,7 @@ in
 
           plasma-desktop
           plasma-workspace
-          plasma-workspace-wallpapers
+          # plasma-workspace-wallpapers
```

Comment:
Why did you comment them out?

Reply:
Ooops- that was meant for another commit.

Reply:
Fixed in latest commit.